Add note about usage reporting time skew limitation (#168)
apollographql/docs: src/content/studio/metrics/usage-reporting.mdx

The request can also optionally include a `Content-Type` header with value `application/protobuf`, but this is not required.

> Reports more than 50 minutes old will be rejected by the reporting endpoint. If you see an error like `Rejecting report from service <your service> with skewed timestamp`, ensure your traces are current and that your timestamp calculations are accurate.

Reviewer: ```suggestion
> ⚠️ **The reporting endpoint rejects reports that are older than 50 minutes.** If you see an error like `Rejecting report from service <your service> with skewed timestamp`, make sure your traces are current and that your timestamp calculations are accurate.
```
